server/sonar-server/src/main/java/org/sonar/server/qualityprofile/RegisterQualityProfiles.java

```diff
@@ -65,45 +67,47 @@
   private final Languages languages;
   private final ActiveRuleIndexer activeRuleIndexer;
   private final QProfileWsSupport qProfileWsSupport;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "qProfileWsSupport" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)